lua/wikis/commons/Widget/NavBox.lua

```diff
@@ -49,13 +47,15 @@ function NavBox:render()
 		or Variables.varDefault('tournament_parent') -- any Page with a HDB
 	)
 
-	-- have to extract so the child doesn't add the header too ...
-	local title = Table.extract(self.props, 'title')
+	local title = NavBoxChild.buildTitleText(props)
```

Comment:
I think this shows that the initial layer should be a child too

Reply:
extracted it to an extra widget